scripts/ebpf_tracing.cmd

```diff
@@ -119,7 +106,7 @@ if "%command%"=="periodic" (
 	)
 
 	@rem Iterate over all the WFP-state files in the 'traceCommittedPath' directory, and delete files overflowing `max_committed_wfp_state_files`.
-	for /f "skip=%max_committed_wfp_state_files% delims=" %%f in ('dir /b /o-d "!traceCommittedPath!\wfpstate*.cab"') do ( del "!traceCommittedPath!\%%f" )
+	for /f "skip=%max_committed_wfp_state_files% delims=" %%f in ('dir /b /o-d "!traceCommittedPath!\wfpstate*.xml"') do ( del "!traceCommittedPath!\%%f" )
```

Comment:
In my recent PR, I renamed max_committed_wfp_state_files. This might cause some issues when merging. You might want to rebase.

Reply:
was merging already, and adding the doc for the new rundowns in you PR